### `test (windows-latest)` failure is pre-existing and unrelated to this PR

```
race() ... rejects when one of the given operations rejects asynchronously first
error: AssertionError: Promise did not reject: resolved to foo
    at test/race.test.ts:44:5
```

**This PR cannot reach that test.** The diff is `AGENTS.md` plus one JSDoc comment block in `lib/resource.ts` — every changed line there begins with ` *`. The `test` task is `deno test --allow-run --allow-read --allow-env --allow-ffi`, with no `--doc`, so JSDoc examples are never compiled or executed.

**The test is timing-dependent.** `test/race.test.ts:37-41`:

```ts
race([
  asyncResolve(10, "foo"),
  asyncReject(5, "bar"),
  asyncReject(15, "baz"),
])
```

`asyncResolve`/`asyncReject` are `yield* sleep(duration)` (`test/suite.ts:29`). The test relies on 5 ms landing before 10 ms. Windows' default timer resolution is ~15.6 ms, so both timers can fire in the same tick and settle in registration order — making `asyncResolve(10, "foo")` win. That is exactly the observed failure ("resolved to foo").

I reran the job to check for flakiness; it failed a second time, which fits a granularity problem rather than a race that is merely occasionally lost.

Worth noting this is the pattern [effectionx's no-sleep-test-sync policy](https://github.com/thefrontside/effectionx/blob/main/.policies/no-sleep-test-sync.md) exists to prevent. Making the ordering deterministic — resolving explicit signals rather than racing sleeps — would fix it, but that felt out of scope for a docs PR. Happy to open a separate one if useful.

--- a/lib/resource.ts
+++ b/lib/resource.ts
@@ -25,7 +25,6 @@ import { useScope } from "./scope.ts";
  *     let socket = new WebSocket(url);
  *     yield* once(socket, 'open');
  *
- *     // teardown that needs `yield*` goes in ensure(), not finally
  *     yield* ensure(function*() {
  *       socket.close();
  *       yield* once(socket, 'close');

